src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+
+    private double height;
+
+    public RightTriangle(Colors color, double side, double height) {
```

Comment:
Don't pass expressions in the constructor. Create separate variables and pass them on for better code readability. Also, consider whether 'side' should be a property of the 'RightTriangle' class, as it is not mentioned in the checklist.